### DISCUSSION TITLE
Welcome to genie Discussions!

### DISCUSSION THREAD

**Discussion by cerebrate** [arkane-systems/genie#97] [General]
## 👋 Welcome!
  We’re using Discussions as a place to ask questions that aren't necessarily issues, share tips and tricks, and hold other discussions related to the present and future of genie. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by PavelSosin-320** [comment]
The idea - provide links to the relevant documentation related to systemd. Indeed, systemd is not simple and tightly integrated with Kernel, Control group, etc. To use systemd for Linux services management, CG management correctly user needs knowledge spread over numerous Wiki and Man pages. Today Genie documentation doesn't provide even the link to the [Systemd main page](https://systemd.io/) systemd.io Web page explaining the systemd concept. and [Desktop environment](https://www.freedesktop.org/software/systemd/man/environment.d.html#) explains the meaning of environment variables.
The bullet list of utilities used in the application development and troubleshooting can help users too and save time on searching in Google   

- systemctl
- systemd-cgls
- systemd-run
- systemd-analyze
- journalctl
- 
........ numerous.
